### ISSUE TITLE
The problem of no sound and no channels running on webgl，when I replace the bank files with my project.

### ISSUE THREAD

**Issue by LaoLeo** [fmod/Unity-HTML5-Demo#4]
Hi, I have trouble when I replace the bank files with mine project to build on webgl，and that was no sound and no channels running.

![image](https://user-images.githubusercontent.com/18170373/209654276-a07803c2-8251-4db2-850d-5f61d658c050.png)

I had change the code to load my bank file, and no error on webgl runtime. Also, I had  verified that the code works properly on Editor：

![image](https://user-images.githubusercontent.com/18170373/209654686-e034c414-729f-4184-8c22-d28ee98bb316.png)

I think the cause is not in the code. It may be in the bank file or the build parameters of fmod studio.

Thanks for your suggestions! ❤️

**Comment by Connor-FMOD** [comment]
Hi, 

The [FMOD Forums](https://qa.fmod.com) is the best place to go to when looking for help. 

Unfortunately, I was not able to reproduce the issue. 

What version of FMOD and Unity are you using and can you share the code you changed to load your banks?

**Comment by LaoLeo** [comment]
Thanks for your reply. @Connor-FMOD 

Yesterday I found the cause of the problem.

The bank files of my project uses encryption,  which is incompatible with the loadBankMemory API,  and the loadFromWeb function on WebGL just calls loadBankMemory.

I found this from FMOD Docs - [13.13 Cannot FMOD_OPENMEMORY_POINT encrypted data](https://fmod.com/docs/2.02/unity/troubleshooting.html#cannot-fmod_openmemory_point-encrypted-data).

Thanks for your time again!❤️
